Makefile

```diff
@@ -4,17 +4,19 @@ ELASTIC_SOURCE_DIR=../elastic-search
 QUEUE_SOURCE_DIR=../queue
 BUILD_DIR=./build/api
 DEPLOY_DIR=./website
-PHP8=php
-PHP7=php7
+PHP=php
+PHP_COMPOSER=$(PHP)
 COMPOSER=$(PWD)/composer.phar
 
 .PHONY: clean help
-.PHONY: build-all
+.PHONY: build-php74
+.PHONY: build-php81
 .PHONY: build-active-and-missing
 .ALL: help
 
 # Versions that can be built.
-CAKEPHP_VERSIONS = 3.0 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8 3.9 3.10 4.0 4.1 4.2 4.3 4.next 5.0
+CAKEPHP_PHP74_VERSIONS = 3.0 3.1 3.2 3.3 3.4 3.5 3.6 3.7 3.8 3.9 3.10 4.0 4.1 4.2 4.3 4.next
+CAKEPHP_PHP81_VERSIONS = 5.0
```

Comment:
We could split this on cake3/4 and cake5. We'll eventually need this split for Chronos. What should the naming convention be? 

The api servers would be easier to maintain with a clean major version split.

Reply:
Can we group targets by the php versions they need?

Reply:
Do you mean the `build-cakephp-version` targets? I wasn't sure how to do this. We need to specify a config file based on the major release for cakephp 3 and 4 and eventually 5.

Reply:
I was thinking more that we would have different dockerfiles for each 'block' of the API and push each docker file to its own container in dokku and use the `site-api` proxy to glue all the sites together again.

Reply:
If you're open to configuring more dokku servers, that would work. The php version split should be along major releases.

Reply:
I don't mind adding more dokku apps. We have a bunch now and the nginx that dokku spawns is pretty lightweight.

Reply:
I split up the Dockerfiles based on major releases and created separate deploy configs for each. The deploy configs are all copies of the original still until we have the dokku stuff setup.

Reply:
Ended up rolling back that change and creating a single, multi-stage Dockerfile with different php builds stages.

Reply:
Looks good. A multistage build is a clever solution.